pkg/cmd/pkg/package_uninstall.go

```diff
@@ -13,8 +27,69 @@ func newCmdPackageUninstall(ctx api.Context) *cobra.Command {
 		Use:   "uninstall <package-name>",
 		Short: "Uninstall a package",
 		Args:  cobra.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			appIDSet := cmd.Flags().Changed("app-id")
+
+			if cliOnly && (appOnly || appIDSet || all) {
+				return errors.New("--cli cannot be used with --app, --app-id, or --all")
+			}
+			return nil
+		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return invokePythonCLI(ctx)
+			packageName := args[0]
+
+			if cliOnly {
+				return removeCLI(ctx, packageName)
+			}
+
+			c, err := cosmos.NewClient(ctx, pluginutil.HTTPClient(""))
+			if err != nil {
+				return err
+			}
+
+			installed, err := packageInstallCount(c, packageName)
+			if err != nil {
+				return err
+			}
+			if installed == 0 {
+				return fmt.Errorf("Package '%s' is not installed", packageName)
+			}
+
+			name := packageName
+			removingApp := cmd.Flags().Changed("app-id")
+			if removingApp {
+				name = appID
+			}
+
+			fmt.Fprintln(ctx.Out(), warningMessage(name, all))
+			promptMsg := promptMessage(name, removingApp, all)
+
+			expected := name
+			if all {
+				expected = fmt.Sprintf("uninstall all %s", name)
+			}
+			if !uninstallConfirmed(ctx, expected, promptMsg, yes) {
+				return errors.New("cancelling uninstall")
+			}
+
+			resp, err := c.PackageUninstall(packageName, all, appID)
+			if err != nil {
+				return err
+			}
+			for _, r := range resp.Results {
+				fmt.Fprintf(ctx.Out(), "Uninstalled package [%s] version [%s]\n", r.PackageName, r.PackageVersion)
+				fmt.Fprintln(ctx.Out(), r.PostUninstallNotes)
```

Comment:
These informational messages should go to stderr (see https://github.com/dcos/dcos-cli/blob/master/design/style.md#stdoutstderr)